internal/database/list.go

```diff
@@ -45,6 +45,7 @@ func (r *ListRepo) List(ctx context.Context) ([]*domain.List, error) {
 		"include_unmonitored",
 		"include_alternate_titles",
 		"skip_clean_sanitize",
+		"include_year",
```

Comment:
Minor nickpick but can we move this up above skip so the includes are next to each others? 😄